tests/ft_erlang.vader

```diff
@@ -0,0 +1,95 @@
+Include: include/setup.vader
+
+Before (prepare an empty working directory):
+  let s:root = tempname()
+  call mkdir(s:root, 'p')
+  let s:old_cwd = fnamemodify('.', ':p')
+  execute 'cd ' . s:root
+
+After (clean up the working directory):
+  execute 'cd ' . s:old_cwd
+  execute 'silent !rm -rf ' . s:root
```

Comment:
Unfortunately Vader does not support nested Before/After, and there is a global Before/After already that should not be overwritten.
I have to think about how to fix this best.
Apart from that, the `rm -rf` looks a bit scary and is not necessary, since Vim should do the cleanup.